### ISSUE TITLE
PowerShell 7.0.0.-rc.2 - Run As Administrator - First keystroke inserting '@' symbol

### ISSUE THREAD

**Issue by GitProjecc** [PowerShell/PowerShell#11828]
Launched PWSH as administrator (from start menu entry - recently added) and ran several commands:
$PSVersionTable
$PSVersionTable | fl *
$env:POWERSHELL_DISTRIBUTION_CHANNEL
$env:PSModulePath
gci 'C:\Program Files\PowerShell\'
gci 'C:\Program Files\PowerShell\7-preview\'
gci 'C:\Program Files\PowerShell\7-preview\' -Filter *exe
ls C:\Users\\$User\AppData\Roaming\code
sl C:\Users\\$User\AppData\Roaming\code
ls .\User\\
ls C:\Users\\$User\scripts\
ls C:\Users\\$User\scripts\psscripts\
ls C:\Users\\$User\scripts\psscripts\ -Directory

At some point during these commands I focused a different window (non-shell) and then re-focused the console. When I started typing commands, it would insert an @ symbol on the first keystroke instead of the chosen character (even if Shift or Enter was pressed). I was able to repeat this behavior inside of the problem console, but was unable to repeat the behavior in new consoles launched as Administrator and I did not see this behavior at all when launching without administrator privileges. The problem also did not appear when launching the shell inside of a VSCode session as administrator.

During my commands, I did use tab completion for the $env: cmdlets by holding down the tab button cycling through all the iterations, and also did shift-tab to reverse the order. It seems super obscure, and I am not sure how to reproduce this behavior, but wanted to make you aware.

Name                           Value
----                           -----
PSVersion                      7.0.0-rc.2
PSEdition                      Core
GitCommitId                    7.0.0-rc.2
OS                             Microsoft Windows 10.0.18362
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

**Comment by vexx32** [comment]
@GitProjecc can you install / update to the latest prerelease version of PSReadLine (if you're not on that version already) and see if the issue still occurs for you? 🙂

**Comment by GitProjecc** [comment]
@vexx32 I did just update my PSReadline since your suggestion. However, I was unable to replicate this behavior outside of the console that I first noticed the issue in, even after launching other consoles (both as admin and regular user) whether launching PWSH standalone, or inside of VS Code. 

I am sure this behavior is hard to replicate since I could not replicate it myself after several different attempts. I just wanted to make sure to bring it up in case someone else does experience the same behavior that I observed.
